### ISSUE TITLE
Comparable across batches

### ISSUE THREAD

**Issue by baigal628** [ventolab/CellphoneDB#101]
Hi,

Thank you for developing this nice tool. I have mouse data from different time points and strains. I ran through them via cellphoneDB individually. I wonder if the output means and p-values are comparable across different runs.

Thanks,
Gali

**Comment by luzgaral** [comment]
Dear @baigal628 

Thank you for using CellphoneDB. 

What do you mean by different runs?

P-values should be comparable as long as your background populations are comparable. 
However, expression means may be susceptible to batch effects (unless you correct the expression counts directly).

Best

Luz

**Comment by baigal628** [comment]
Hi,

Thanks for the reply. It makes sense to me. I have another question. When I ran on a large sample, I realized that not all the cell types in the input meta file are included in the pvalue.txt/mean.txt. I assume all the cell-cell interactions will be reported in these two files even if they are not significant, right? For example, if I have 36 cell types in my meta file, I expect I have 36x36 cell-cell interactions as the output. Is it possibly due to the downsampling size being too small so that they are not captured? Thanks, Gali
